website/server/controllers/api-v3/tasks.js

```diff
@@ -611,6 +611,111 @@ api.scoreTask = {
   },
 };
 
+api.scoreTasks = {
+  method: 'POST',
+  url: '/tasks/score',
+  middlewares: [authWithHeaders()],
+  async handler (req, res) {
+    req.checkBody('tasks', res.t('tasksRequired')).notEmpty();
+
+    let user = res.locals.user;
+    let resJsonData = [];
+
+    for (let i = 0; i < req.body.tasks.length; i++) {
```

Comment:
all this code is mostly copied from the scoreTask route, it should be abstracted to avoid duplication